### PR TITLE
Style table visualization

### PR DESCRIPTION
**Pull request by axelboc-esrf**
Screenshot from 2020-03-13 17-03-53

### CODE REVIEW

File: src/h5web/dataset-visualizer/DatasetVisualizer.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ function DatasetVisualizer(props: Props): JSX.Element {
       isBaseType(type) &&
       isSimpleShape(shape) &&
       [1, 2].includes(shape.dims.length) ? (
-        <TableVis dims={shape.dims} data={value} />
+        <MatrixVis dims={shape.dims} data={value} />
```
Comment (loichuder) [inline src/h5web/dataset-visualizer/DatasetVisualizer.tsx]: I would advise to add "renaming to Matrix" in the pull request/issue name

**Comment by andygotz** [comment]
This sounds interesting - how can I visualize the results?

**Comment by vasole** [comment]
@andygotz I guess you can try

https://h5web.netlify.com/

**Comment by axelboc** [comment]
> This sounds interesting - how can I visualize the results?

If you look under "All checks have passed", and click on the "Details" link of the last item ("netlify/h5web/deploy-preview"), you'll be redirected to the PR's deploy preview.

I'm merging now anyway, but still good to know for future PRs. ;)

**Comment by axelboc** [comment]
Arf, should have uploaded a screenshot... Anyway, alternatively, you can always access Deploy Previews at URLs of the form https://deploy-preview-23--h5web.netlify.com/, where 23 is the number of the PR.

**Pull request merged**